### PR TITLE
[TRANSLATION] Whoami Polish translation

### PR DESCRIPTION
**Pull request by Mcpg**
## Purpose

Translated `whoami` cmd utility to Polish.

### CODE REVIEW

**Comment by Mcpg** [comment]
OK, after a few changes, I think the translation is ready.

File: base/applications/cmdutils/whoami/lang/pl-PL.rc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    IDS_ERROR_VALUENOTALLOWED "BŁĄD: Nieprawidłowa składnia. Wartość '%s' nie jest dozwolona dla opcji '/fo'.\nWpisz polecenie ""WHOAMI /?"", aby zobaczyć sposób użycia.\n"
+    IDS_ERROR_1TIMES "BŁĄD: Nieprawidłowa składnia. Opcja '%s' nie jest dozwolona więcej niż '1' raz(y).\nWpisz polecenie ""WHOAMI /?"", aby zobaczyć sposób użycia.\n"
+    IDS_ERROR_INVALIDSYNTAX "BŁĄD: Nieprawidłowa składnia.\nWpisz polecenie ""WHOAMI /?"", aby zobaczyć sposób użycia.\n"
+    IDS_ERROR_INVALIDARG "BŁĄD: Nieprawidłowy argument/opcja - '%s'.\nWpisz polecenie ""WHOAMI /?"", aby zobaczyć sposób użycia.\n"
+    IDS_ERROR_NH_LIST "BŁĄD: Przełącznika /NH nie można używać w formacie LIST.\nWpisz polecenie ""WHOAMI /?"", aby zobaczyć sposób użycia.\n"
+    IDS_HELP "OPIS:\n\
+  Wyświetla informacje użytkownika, grupy i uprawnień dla lokalnie zalogowanego użytkownika\n\
+  Jeżeli nie podano żadnych parametrów, wyświetla aktualną domenę i nazwę użytkownika.\n\
+\n\
+  Dostępnymi formatami wyjściowymi dla opcji '/fo\ są 'csv', 'list' i 'table'.\n\
```
Comment (HBelusca) [inline base/applications/cmdutils/whoami/lang/pl-PL.rc]: `opcji '/fo\ są` <---- problem with the closing quote after `/fo`

File: base/applications/cmdutils/whoami/lang/pl-PL.rc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  Użyj opcji '/nh\, żeby ukryć nagłówki. Domyślnie dane są wyświetlone w tabeli.\n\
```
Comment (HBelusca) [inline base/applications/cmdutils/whoami/lang/pl-PL.rc]: `opcji '/nh\,` <---- problem with the closing quote after `/nh`

**Comment by Mcpg** [comment]
Alright, fixed. It should be ready

**Pull request merged**